core/src/main/java/dev/streamx/cli/exception/DockerException.java

```diff
@@ -19,6 +21,18 @@ private DockerException(String message) {
     super(message);
   }
 
+  public static DockerException containerStartupFailed(String message) {
+    return new DockerException("""
```

Comment:
Please use the prompt I've proposed in the ticket. 
+ We should not reference the links (it's hard to maintain link hardcoded in the code)